libbeat/tests/system/beat/compose.py

```diff
@@ -51,10 +51,13 @@ def is_healthy(container):
             return container.inspect()['State']['Health']['Status'] == 'healthy'
 
         project = cls.compose_project()
+        project.build(
+            service_names=cls.COMPOSE_SERVICES,
+            build_args=cls.COMPOSE_BUILD_ARGS)
         project.up(
             strategy=ConvergenceStrategy.always,
             service_names=cls.COMPOSE_SERVICES,
-            do_build=BuildAction.force,
+            do_build=BuildAction.skip,
```

Comment:
Are you sure about this change?

Reply:
Yes, after adding a explicit `project.build` it is not needed to build also in `project.up`. The reason to add an explicit `project.build` is that  `project.up` doesn't allow to pass build arguments.